docs/project_setup_and_build.md

```diff
@@ -0,0 +1,229 @@
+    Dockerfile
+    project_setup.py
+    deployment_config.py
+```
+
+Let us start with the `project_package`. It's the Python package which contains the processing logic of your workflows.
+It also contains `Workflow` objects, which arranges parts of your processing logic into 
+a [DAG](https://en.wikipedia.org/wiki/Directed_acyclic_graph) (read the [Workflow & Job chapter](./workflow-and-job.md) to learn more about workflows and jobs).
+The `project_package` is used to create a standard Python package that can be installed using `pip`.
+
+`project_setup.py` is the build script for the project. It turns the `project_package` into a `.whl` package. 
+It's based on the standard Python tool  — [setuptool](https://packaging.python.org/key_projects/#setuptools).
+
+There is also the special variable  — `PROJECT_NAME` inside `project_setup.py`. In the example project, it is 
+`PROJECT_NAME = 'project_package'`. It tells BigFlow CLI which package inside the `project_dir` is the main package with
+your processing logic and workflows.
+
+You can put your tests into the `test` package. The `bigflow build-package` command runs tests automatically, before trying to build the package.
+
+The `resources` directory contains non-Python files. That is the only directory that will be packaged
+along the `project_package` (so you can access these files after installation, from the `project_package`). Any other files
+inside the `project_directory` won't be available for the `project_package`. `resources` can't be nested. So you can't
+have a directory inside the `resources` directory.
+
+The `get_resource_absolute_path` function allows you to access files from the `resources` directory, being inside the `project_package`.
+
+[**`resources.py`**](examples/project_structure_and_build/resources_workflow.py)
+```python
+with open(get_resource_absolute_path('example_resource.txt', Path(__file__))) as f:
+    print(f.read())
+```
+
+Run the example above, using the following command:
+
+```shell script
+bigflow run --workflow resources_workflow
+```
+
+Result:
+
+```
+Welcome inside the example resource!
+```
+
+The two remaining files  — `Dockerfile` and `deployment_config.py` don't take a part in the Python package build process.
+
+Because a BigFlow project is mainly a standard Python package, we suggest going through the 
+[official Python packaging tutorial](https://packaging.python.org/tutorials/packaging-projects/).
+
+### Package builder
+
+The `bigflow build-package` command takes 3 steps to build a Python package from your project:
+
+1. Cleans leftovers from a previous build.
+1. Runs tests from the `test` package and generates JUnit xml report, 
+using the [`unittest-xml-reporting`](https://pypi.org/project/unittest-xml-reporting/) package. You can find the generated report
+inside the `project_dir/build/junit-reports` directory.
+1. Finally, `bigflow build-package` runs the `bdist_wheel` setup tools command. It generates a `.whl` package that you can
```

Comment:
Skoro wyliczamy, to może nie warto powtarzać "Finally, `bigflow build-package` runs" tylko: 
1. Runs the `bdist_wheel` setup tools command.

no i "package that" -> "package which"

Reply:
done